### PR TITLE
Malf Fix

### PR DESCRIPTION
**Pull request by ghost**
Fixes the fact that you can't hack your own borgs with the robo console because special_role is not set. Who knows, maybe it does get set but not with adminbus, and we never get malf outside of adminbus. Borgs can't hack themselves like they can before but like, why would they need to anyway?

### CODE REVIEW

**Comment by SonixApache** [comment]
that's one outta three, good start #23072

**Comment by ghost** [comment]
Malfs being unable to recall is a feature boss. And I couldn't replicate the borg one, there's absolutely no way they should be able to get the nuke spell.

**Comment by SonixApache** [comment]
Pre-dynamic, the shuttle would auto-recall at 5 mikes remaining. I'd say that's feature loss.
I'll do some checking on multi-nuke, it was still an issue when I made that report.

File: code/game/machinery/computer/robot.dm [hunk 1]
```diff
@@ -75,9 +75,8 @@
 					dat += " Slaved to [R.connected_ai.name] |"
 				else
 					dat += " Independent from AI |"
-				if (istype(user, /mob/living/silicon))
-					if((user.mind.special_role && user.mind.original == user) && !R.emagged)
-						dat += "<A href='?src=\ref[src];magbot=\ref[R]'>(<font color=blue><i>Hack</i></font>)</A> "
+				if(istype(user, /mob/living/silicon) && locate(/datum/role/malfAI) in user.mind.antag_roles && !R.emagged)
```
Comment (jknpj) [inline code/game/machinery/computer/robot.dm]: issilicon(user)
ismalf(user)

**Comment by SonixApache** [comment]
merged with no 🆑 
😔

**Pull request merged**